integrations-service/integrations/utils/integrations/remote_browser.py

```diff
@@ -23,6 +23,16 @@
 
 CURSOR_PATH = Path(__file__).parent / "assets" / "cursor-small.png"
 
+# Add connection pooling
+_browser_pool = {}
+
+
+async def get_browser(connect_url: str):
```

Comment:
We need to cleanup old/disconnected browsers & limit pool size

Reply:
Yet to test it out.

Reply:
Is this really needed?

Reply:
this can be problematic. if we have multiple workers, they wont be sharing the state and the browser pool, some other worker might run the browser.close() method on the same browserbase instance when hitting the max_pool_size limit

Reply:
I have removed the browser pool limit from this PR and will create a new one